crates/common/fmt/src/ui.rs

```diff
@@ -808,6 +817,16 @@ impl UIfmt for EthValue {
     }
 }
 
+impl UIfmt for SignedAuthorization {
+    fn pretty(&self) -> String {
+        let signed_authorization = serde_json::to_string(self).unwrap();
+        let recover_authority = self.recover_authority().unwrap();
```

Comment:
this can fail because these are allowed to be invalid.

we can use "invalid" if this is the case

Reply:
Updated, pls check again. @mattsse